modules/helpers/token.js

```diff
@@ -0,0 +1,144 @@
+export function registerTokenControls() {
+  game.settings.register("starwarsffg", "showMinionCount", {
+    name: game.i18n.localize("SWFFG.Settings.showMinionCount.Name"),
+    hint: game.i18n.localize("SWFFG.Settings.showMinionCount.Hint"),
+    scope: "world",
+    config: true,
+    default: true,
+    type: Boolean,
+    onChange: (rule) => window.location.reload()
+  });
+  game.settings.register("starwarsffg", "showAdversaryCount", {
+    name: game.i18n.localize("SWFFG.Settings.showAdversaryCount.Name"),
+    hint: game.i18n.localize("SWFFG.Settings.showAdversaryCount.Hint"),
+    scope: "world",
+    config: true,
+    default: true,
+    type: Boolean,
+    onChange: (rule) => window.location.reload()
+  });
+    game.settings.register("starwarsffg", "adversaryItemName", {
+    name: game.i18n.localize("SWFFG.Settings.AdversaryItemName.Name"),
+    hint: game.i18n.localize("SWFFG.Settings.AdversaryItemName.Hint"),
+    scope: "world",
+    config: true,
+    default: "Adversary",
+    type: String,
+    onChange: (rule) => window.location.reload()
+  });
+}
+
+export function drawMinionCount(token) {
+  if (!game.settings.get("starwarsffg", "showMinionCount")) {
+    return;
+  }
+  const borderWidth = 0.35;
+  const friendlyColor = "0x00A2E84D";
+  const enemyColor = "0x8800154D";
+  const overflowColor = "0xDAA520";
```

Comment:
For accessibility / customisability these sorts of things would probably be better as CSS variables, or system settings, so that other modules can interact with them, and users can easily tweak them. But that's a long term concern, not a "get it working" concern.